### DISCUSSION TITLE
local external embedding of a page

### DISCUSSION THREAD

**Discussion by victor-zorro** [requarks/wiki#6418] [Help / Questions] [answered]
Hi There,

I like WikiJS and we use it at our office to share manuals and protocols and information about how we work.
Like a proper intranet. What I would like, is to create a WikiJS page, and to embed another page, running locally.
Eg a node-red dashboard. So I can look at it from the outside world.

WikiJS runs on our office fileserver, as is node-red.
Reason I want to do this, is to maintain one location people login to.

I assume that this is not possible easy, but who knows....

Thanks

Vincent

**Comment by NGPixel** [comment, marked as answer]
Use an iframe. Just make sure you enable iframe under Admin > Rendering > HTML > Security.
